### DISCUSSION TITLE
Resize plugin: `module.esm.min.js` build contains intersect plugin code

### DISCUSSION THREAD

**Discussion by mgschoen** [alpinejs/alpine#4412] [7. Bugs (archived)]
In the latest published version of the Resize plugin, the minified ESM build contains the code for the Intersect plugin instead of the Resize plugin. All other builds including the non-minified ESM build are fine, as far as I can tell.

See for example: https://cdn.jsdelivr.net/npm/@alpinejs/resize@latest/dist/module.esm.min.js

**Comment by ekwoka** [comment]
that is hilarious

Yeah the package info is messed up, but then that one is wrong...so strange

@calebporzio the build code works correctly, I think you just had some local file issues when you published. Though updating the homepage url for the resize plugin could be good.
